code/__HELPERS/_lists.dm

```diff
@@ -472,15 +472,15 @@
 			return //no need to move
 		from_index += len //we want to shift left instead of right
 
-		for(var/i = 0, i < distance, ++i)
+		for(var/i in 1 to distance)
```

Comment:
This is wrong in the same vein, but I wonder if the old code was just always wrong?

Reply:
This one is not wrong. `i` starts at 0 here.